### PR TITLE
ts/bullet-series

### PR DESCRIPTION
**Pull request by marvin19**
Migration of bullet series.

### CODE REVIEW

File: ts/Series/Bullet/BulletSeries.ts [hunk 1]
```diff
@@ -0,0 +1,511 @@
...
+        if (this.targetGraphic) {
+            this.targetGraphic = this.targetGraphic.destroy() as any;
+        }
+        super.destroy.apply(this, arguments);
+        return;
+    }
+
+    /* eslint-enable valid-jsdoc */
+
+}
```
Comment (bre1470) [inline ts/Series/Bullet/BulletSeries.ts:392]: This should go into a `BulletPoint.ts` (instead of the `BulletPoint.d.ts`).

**Review by bre1470 — CHANGES_REQUESTED** [review]
Thank you. One thing with the point class can be improved.

**Review by bre1470 — APPROVED** [review]
Thank you! Looks great. 👍

**Comment by bre1470** [comment]
There is a visual test differing.

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
Pending a fix for the visual diff. Apparently the `target` value is no longer considered.

**Comment by bre1470** [comment]
We unable to reproduce it locally. Additionally the graph looks completely different in the browser.

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**